tests/unit/fetch-test.js

```diff
@@ -0,0 +1,25 @@
+var simple = require('simple-mock')
+var test = require('tape')
+
+var fetch = require('../../lib/fetch')
+var internals = fetch.internals
+
+test('fetch', function (t) {
+  simple.mock(internals, 'fetchProperties').resolveWith({})
```

Comment:
that’s what we can do to simulate a successful server response, instead of mocking low level http APIs